<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cuerpo</class>
 <widget class="QWidget" name="cuerpo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="inf_isq">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>328</y>
     <width>54</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/imagenes/llantaI.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="grafico">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>222</width>
     <height>222</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;

</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="inf_der">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>328</y>
     <width>54</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/imagenes/llantaD.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="sup_der">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>16</y>
     <width>54</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/imagenes/llantaD.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cuerpo_2">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>16</y>
     <width>290</width>
     <height>415</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/imagenes/Interfaz.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sup_isq">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>16</y>
     <width>54</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/imagenes/llantaI.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="izq_sup">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="dere_sup">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="izq_inf">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="dere_inf">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
